packages/backend-app-api/src/alpha/featureDiscoveryServiceFactory.ts

```diff
@@ -165,6 +169,38 @@ export const featureDiscoveryServiceFactory = createServiceFactory({
   },
 });
 
+/**
+ * A loader that discovers backend features from the current package.json and its dependencies.
+ *
+ * @public
+ *
+ * @example
+ * Using the `featureDiscoveryLoader` loader in a backend instance:
+ * ```ts
+ * //...
+ * import { createBackend } from '@backstage/backend-defaults';
+ * import { featureDiscoveryLoader } from '@backstage/backend-app-api';
+ *
+ * const backend = createBackend();
+ * backend.add(featureDiscoveryLoader);
+ * //...
+ * backend.start();
+ * ```
+ */
+export const featureDiscoveryLoader = createBackendFeatureLoader({
```

Comment:
Should this be public?
It is automatically called in the `BackendInitializer`, so I was unsure if I should export this.

Reply:
Naming: `discoveryFeatureLoader` perhaps? Or `dependencyFeatureLoader` so as not to confuse with the discovery service?

Reply:
Renamed to `discoveryFeatureLoader ` 🙂